lib/presenter/Chart.js

```diff
@@ -34,7 +34,7 @@ class Chart extends Events {
             {top: 0, left: 100, bottom: 50, right: 10 },
             (options.margins || {})
         );
-        this.padding       = (typeof options.padding === 'number') ? options.padding : 10;
+        this.padding       = (typeof options.padding.top === 'number') ? options.padding : {top: 10, bottom: 10, left: 10, right: 10};
```

Comment:
Maybe do an extend here? (aka `Object.assign` -- check out `this.margins` above)